Sources/AccountSDKIOSWeb/Simplified Login/SimplifiedLoginUIFactory.swift

```diff
@@ -78,6 +78,29 @@ struct SimplifiedLoginUIFactory {
             nc.pushViewController(webVC, animated: true)
         }
         
+        viewModel.onClickedContinueAsUser = {
+            assertionFetcher.fetchAssertion { result in
+                switch result {
+                case .success(let assertion):
+                    DispatchQueue.main.async {
+                        let session = client.createWebAuthenticationSession(withMFA: nil, loginHint: nil, assertion: assertion.assertion, extraScopeValues: [], completion: completion)
+                        viewModel.asWebAuthenticationSession = session
+                        
+                        if #available(iOS 13.0, *) {
+                            let context = ASWebAuthSessionContextProvider()
+                            session.presentationContextProvider = context //TODO: Perhaps should be passed in
+                            session.prefersEphemeralWebBrowserSession = true
+                        }
+                        
+                        session.start()
+                    }
+                case .failure(let error):
+                    // TODO: How should we fail gracefully here
```

Comment:
Pop-up with a meaningful error for user should be good. But what next? hide simplified login?